README.md

```diff
@@ -64,7 +64,8 @@ class DateTimeForm extends React.Component {
           placeholder="Date Time"
           value={this.state.dateTime}
           iconPosition="left"
-          onChange={this.handleChange} />
+          onChange={this.handleChange} 
+          preserveViewMode={true} />
```

Comment:
This is redundant because component has ``preserveViewMode: true`` in default props.